Upstream patch:

commit ff889b196575c2fbf6aa7130abb1ec862714ea4e
Author: Florian Weimer <fweimer@redhat.com>
Date:   Fri Feb 19 14:21:34 2016 +0100

    Remove trailing newline from date_fmt in Serbian locales [BZ #19581]

diff --git a/localedata/locales/sr_ME b/localedata/locales/sr_ME
index 4f243dc..dd68df8 100644
--- a/localedata/locales/sr_ME
+++ b/localedata/locales/sr_ME
@@ -119,7 +119,7 @@ am_pm   "";""
 t_fmt_ampm "<U0025><U0054>"
 date_fmt  "<U0025><U0061><U002c><U0020><U0025><U0065><U002E><U0020>/
 <U0025><U0062><U0020><U0025><U0059><U002E><U0020><U0020><U0025><U0048>/
-<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A><U000A>"
+<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A>"
 week    7;19971130;4
 first_weekday 2
 first_workday 2
diff --git a/localedata/locales/sr_RS b/localedata/locales/sr_RS
index 2ae085b..ffea86f 100644
--- a/localedata/locales/sr_RS
+++ b/localedata/locales/sr_RS
@@ -300,7 +300,7 @@ am_pm   "";""
 t_fmt_ampm "<U0025><U0054>"
 date_fmt  "<U0025><U0061><U002C><U0020><U0025><U0065><U002E><U0020>/
 <U0025><U0062><U0020><U0025><U0059><U002E><U0020><U0020><U0025><U0048>/
-<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A><U000A>"
+<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A>"
 week    7;19971130;4
 first_weekday 2
 first_workday 2
diff --git a/localedata/locales/sr_RS@latin b/localedata/locales/sr_RS@latin
index da6628b..fd10ea6 100644
--- a/localedata/locales/sr_RS@latin
+++ b/localedata/locales/sr_RS@latin
@@ -120,7 +120,7 @@ am_pm   "";""
 t_fmt_ampm "<U0025><U0054>"
 date_fmt  "<U0025><U0061><U002c><U0020><U0025><U0065><U002E><U0020>/
 <U0025><U0062><U0020><U0025><U0059><U002E><U0020><U0020><U0025><U0048>/
-<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A><U000A>"
+<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A>"
 week    7;19971130;4
 first_weekday 2
 first_workday 2
